runbotics/runbotics-orchestrator-ui/src/views/process/ProcessBuildView/Modeler/ConfigureActionPanel/widgets/BrowserTargetWidget.tsx

```diff
@@ -0,0 +1,105 @@
+import React, { FC, useEffect } from 'react';
+
+import { InputLabel, MenuItem, Select, SelectChangeEvent } from '@mui/material';
+import { WidgetProps } from '@rjsf/core';
+import styled from 'styled-components';
+
+import { translate } from 'src/hooks/useTranslations';
+
+import ElementAwareAutocompleteWidget from './ElementAwareAutocompleteWidget';
+
+const FieldsWrapper = styled.div`
```

Comment:
move everything but component to appropriate files like `.types`, `.utils`, `.styles`